server/routes/cbauthentication/__tests__/cb_register.test.js

```diff
@@ -16,6 +16,7 @@ test('POST /api/cb/register | viable & registered CB', async (t) => {
     orgName: 'Slack Jawed and Dopamine',
     email: 'jinglis12@googlemail.com',
     category: 'Housing',
+    region: 'Wales',
```

Comment:
"Wales" is not part of the seeds for the regions in the API.

Reply:
https://github.com/TwinePlatform/twine-visitor/blob/b47400f2e2c00fffb3892834b92918bb7acbf595/client/src/cb-admin/pages/Signup.js#L36-L45

🤷🏽‍♀️